specification/cognitiveservices/OpenAI.Inference/models/whisper.common.tsp

```diff
@@ -0,0 +1,57 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+import "./transcription.create.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("Defines the format of the output.")
+enum AudioTranscriptionFormat {
+    @doc("JSON format. The translation/transcription response will only contain 'text'.")
+    json: "json",
+
+    @doc("Text format. The translation/transcription response will be of type text/plain.")
+    text: "text",
+
+    @doc("SRT format. The translation/transcription response will be of type text/plain.")
+    srt: "srt",
+
+    @doc("Verbose JSON format.The translation/transcription response will contain additional information.")
+    verbose_json: "verbose_json",
+
+    @doc("VTT format. The translation/transcription response will be of type text/plain.")
+    vtt: "vtt",
+}
+
+@doc("Transcription segment.")
+model AudioTranscriptionSegment {
+    @doc("Segment identifier.")
+    id?: string;
+
+    @doc("Segment start offset.")
+    start?: int32;
```

Comment:
I think start and end may be `float32` (second units like duration) -- and we should ask if this is appropriate to represent as an offset or similar.

Reply:
would we need the `encode` decorator here too?